ngc_flowctrl.c

```diff
@@ -689,11 +689,13 @@ status_code_t ngc_flowctrl (uint32_t o_label, char *line, uint_fast8_t *pos, boo
                     else if((g65_return = !!grbl.on_macro_return))
                         ngc_flowctrl_unwind_stack(stack[stack_idx].file);
 
-                    if(ngc_eval_expression(line, pos, &value) == Status_OK) {
+                    if(ngc_read_real_value(line, pos, &value) == Status_OK) {
```

Comment:
ngc_read_real_value will call ngc_eval_expression if the first character encountered is [, so this does not break compatibility with current usage. It only adds functionality 😅